components/layout/ModPage.vue

```diff
@@ -465,6 +467,14 @@ export default {
 
       this.userFollows.splice(this.userFollows.indexOf(this.mod.id), 1)
     },
+    formatTime(date) {
+      date = '2021-05-10T03:24:42.106423Z'
+      let defaultMessage = this.$dayjs(date).fromNow()
+      if (defaultMessage.length > 13) {
+        defaultMessage = defaultMessage.replace('minutes', 'min.')
+      }
+      return defaultMessage
+    },
```

Comment:
No mods were updated recently enough at the time of writing to test this, but, I'm going to assume it works :+1: